modules/network/src/integrationTest/java/org/apache/ignite/internal/network/recovery/ItRecoveryHandshakeTest.java

```diff
@@ -399,6 +400,7 @@ private ConnectionManager startManager(
             ClientStageFail clientHandshakeFailAt
     ) {
         var registry = new TestMessageSerializationRegistryImpl();
+        var serializationService = new SerializationService(registry, null);
```

Comment:
If the second argument is optional, then how about adding another constructor with just first argument? Also, in such a case it makes sense to annotate the second argument as `@Nullable`.

Or it's just for tests for which we are sure that they will not use the second argument?

Reply:
Absolutely sure of the second thing, in these tests marshallable objects are not used

Reply:
Ok